DiffusionPreprocessing/DiffPreprocPipeline.sh

```diff
@@ -399,31 +407,61 @@ get_options()
 	echo "   b0maxbval: ${b0maxbval}"
 	echo "   runcmd: ${runcmd}"
 	echo "   CombineDataFlag: ${CombineDataFlag}"
+	if [ ! -z ${SelectBestB0} ]; then
+		echo "   SelectBestB0: ${SelectBestB0}"
+	fi
 	echo "   extra_eddy_args: ${extra_eddy_args}"
 	echo "-- ${g_script_name}: Specified Command-Line Parameters - End --"
+
+	if [ ! -z "${SelectBestB0}" ]; then
+		dont_peas_set=false
+		fwhm_set=false
+		if [ ! -z "${extra_eddy_args}" ]; then
+			for extra_eddy_arg in ${extra_eddy_args}; do
+				if [[ ${extra_eddy_arg} == "--fwhm"* ]]; then
+					fwhm_set=true
+				fi
+				if [[ ${extra_eddy_arg} == "--dont_peas"* ]]; then
+					show_usage
+					echo -e ""
+					echo -e "ERROR: When using --select-best-b0 post-alignment of shells in eddy is required, "
+					echo -e "       as the first b0 could be taken from anywhere within the diffusion data and "
+					echo -e "       hence might not be aligned to the first diffusion-weighted image."
+					echo -e "       Remove either the --extra_eddy_args=--dont_peas flag or the --select-best-b0 flag"
+					echo ""
+					exit 1
+				fi
+			done
+		fi
+		if [ ${fwhm_set} == false ]; then
+			log_Warn "WARNING: Using --select-best-b0 prepends the best b0 to the start of the file passed into eddy"
```

Comment:
`log_Warn` already appends with "WARNING:".  (Just as `log_Err` already appends with "ERROR:")
```suggestion
			log_Warn "Using --select-best-b0 prepends the best b0 to the start of the file passed into eddy"
```